### PR TITLE
Handle user errors for tester and group actions

### PR DESCRIPTION
**Pull request by tagboola**

### CODE REVIEW

File: lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_add_testers_action.rb [hunk 1]
```diff
@@ -102,6 +93,49 @@ def self.available_options
...
+
+        begin
+          client.batch_project_group_join(group_name(project_number, group_alias), request)
+        rescue Google::Apis::Error => err
+          case err.status_code.to_i
+          when 400
+            UI.user_error!(ErrorMessage::INVALID_EMAIL_ADDRESS)
+          when 404
+            UI.user_error!(ErrorMessage::INVALID_TESTER_GROUP)
+          else
```
Comment (kaibolay) [inline lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_add_testers_action.rb:134]: also check for `TESTER_LIMIT_VIOLATION`

File: lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_delete_group_action.rb [hunk 2]
```diff
@@ -22,7 +22,16 @@ def self.run(params)
 
         UI.message("⏳ Deleting tester group '#{group_alias}' in project #{project_number}...")
 
-        client.delete_project_group(group_name(project_number, group_alias))
+        begin
+          client.delete_project_group(group_name(project_number, group_alias))
+        rescue Google::Apis::Error => err
+          case err.status_code.to_i
+          when 404
```
Comment (kaibolay) [inline lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_delete_group_action.rb:29]: Are you sure this is a 404 and not a 400?

File: lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_add_testers_action.rb (hunk 1 above)
Comment (tagboola) [reply lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_add_testers_action.rb:136]: Done.

File: lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_delete_group_action.rb (hunk 2 above)
Comment (tagboola) [reply lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_delete_group_action.rb:29]: Yep just double checked the backend code.

**Pull request merged**